.gitignore

```diff
@@ -27,3 +27,4 @@ hs_err_pid*
 /grid_report.pdf
 /grid_report.xlsx
```

Comment:
Adding each individual file into .gitignore doesn't seem a good idea in terms of maintainability. 
I would suggest moving all generated files into the same folder (i.e test_reports) and then just exclude the whole folder.
This way, when adding new tests to the suite, no further changes will be required here.

Reply:
Agree, it will be done in the following commit.